docs/DeveloperGuide.md

```diff
@@ -639,13 +639,28 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+### Finding a person
+
+1. Finding with parameters missing.
+   1. Test case: `find` <br>
+      Expected: An error message is shown stating that the command format is incorrect and showing the user the correct format.
+   2. Test case: `find t/` <br>
+      Expected: Listing out everyone is the contact list.  
+2. Finding a person with all possible parameters specified.
+   1. Test case:  `find n/John Doe p/98765432 r/08-0805 t/friends` <br>
+      Expected: Listing out the contact with a name John Doe, with a phone number of 98765432, and lives in the room 08-0805, who has a tag of friends.
```

Comment:
Should this be "Displaying the contact with name John Doe, phone number of 98765432, living in room #08-0805 and has a "friends" tag" instead?